install/kubernetes/cilium/templates/cilium-ingress-service.yaml

```diff
@@ -28,6 +28,9 @@ spec:
   {{- if .Values.ingressController.service.loadBalancerClass }}
   loadBalancerClass: {{ .Values.ingressController.service.loadBalancerClass }}
   {{- end }}
+  allocateLoadBalancerNodePorts: {{ .Values.ingressController.service.allocateLoadBalancerNodePorts }}
```

Comment:
According to [these docs](https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-nodeport-allocation) this field is only stable in v1.24. What happens on older versions of Kubernetes?

Should we only add the key here if the Helm value is explicitly set to true or false? (i.e. via `hasKey`)?

Reply:
good catch 💯, let me make one update, thanks a lot.

Reply:
thanks :+1: